docs/api/ref/api-v3.yml

```diff
@@ -37,14 +37,14 @@ paths:
           explode: false
           schema:
             type: string
-            example: '3017620422003'
-        - $ref: '#/components/parameters/cc'
-        - $ref: '#/components/parameters/lc'
+            example: "3017620422003"
+        - $ref: "#/components/parameters/cc"
+        - $ref: "#/components/parameters/lc"
         - schema:
             type: string
           in: query
           name: tags_lc
-          description: '2 letter language code to request names of tags in a specific language.  For READ requets: if passed, all taxonomized tags of the response will include a lc_name property with the translation in the requested language, if available. Otherwise, the property value will contain the name in the original language, prefixed by the 2 language code and a colon.'
+          description: "2 letter language code to request names of tags in a specific language.  For READ requets: if passed, all taxonomized tags of the response will include a lc_name property with the translation in the requested language, if available. Otherwise, the property value will contain the name in the original language, prefixed by the 2 language code and a colon."
```

Comment:
```suggestion
          description: "2 letter language code to request names of tags in a specific language.  For READ requests: if passed, all taxonomized tags of the response will include a `lc_name` property with the translation in the requested language, if available. Otherwise, the property value will contain the name in the original language, prefixed by the 2 language code and a colon."
```